lib/spack/spack/main.py

```diff
@@ -314,6 +314,12 @@ def make_argument_parser(**kwargs):
                         help="run spack under the pdb debugger")
     parser.add_argument('-k', '--insecure', action='store_true',
                         help="do not check ssl certificates when downloading")
+    parser.add_argument('-l', '--enable-locks', action='store_true',
+                        dest='locks', default=None,
+                        help="force spack to use filesystem locking")
+    parser.add_argument('-L', '--disable-locks', action='store_true',
+                        dest='locks',
+                        help="force spack to disable filesystem locking")
```

Comment:
I think there's an error here. Shouldn't the second argument `store_false`, i.e. :
```python
parser.add_argument('-L', '--disable-locks', action='store_false',
                    dest='locks',
                    help="force spack to disable filesystem locking")
```
?

Reply:
Well that's an important catch.  Thanks!